source/index.html

```diff
@@ -1,12 +1,185 @@
+              деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу
+              сбросить лишний вес.
+            </p>
+            <a class="food-category__link" href="#">
+              каталог slim
+              <img class="food-category__link-image" src="#" alt="#" />
+            </a>
+          </div>
+          <div class="food-category__card card">
+            <h3 class="food-category__title">Набор массы</h3>
+            <img class="food-category__image" src="#" alt="#" />
+            <p class="food-category__text">
+              Заработать авторитет среди дворовых котов и даже собак? Серия Cat
+              Energy Pro поможет вашему коту нарастить необходимые мышцы!
+            </p>
+            <a class="food-category__link" href="#">
+              каталог pro
+              <img class="food-category__link-image" src="#" alt="#" />
+            </a>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="about__wrapper">
+          <h2 class="about__title">Как это работает</h2>
+          <ul class="about__list">
+            <li class="about__item">
+              <img class="about__image" src="#" alt="#" />
+              <p class="about__text">
+                Функциональное питание содержит только полезные питательные
+                вещества.
+              </p>
+            </li>
+            <li class="about__item">
+              <img class="about__image" src="#" alt="#" />
+              <p class="about__text">
+                Выпускается в виде порошка, который нужно лишь залить кипятком и
+                готово.
+              </p>
+            </li>
+            <li class="about__item">
+              <img class="about__image" src="#" alt="#" />
+              <p class="about__text">
+                Замените один-два приема обычной еды на наше функциональное
+                питание.
+              </p>
+            </li>
+            <li class="about__item">
+              <img class="about__image" src="#" alt="#" />
+              <p class="about__text">
+                Уже через месяц наслаждайтесь изменениями к лучшему вашего
+                питомца!
+              </p>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="live-example">
+        <div class="live-example__wrapper">
+          <h2 class="live-example__title">Живой пример</h2>
+          <p class="live-example__text">
+            Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на
+            Cat Energy Slim. Отличный результат без изнуряющих тренировок! При
+            этом он не менял своих привычек и по-прежнему спит по 16 часов в
+            день.
+          </p>
+          <div class="live-example-plate">
+            <div class="live-example-plate__item">
+              <p class="live-example-plate__value"><b>5 КГ</b></p>
+              <p class="live-example-plate__text plate--text">снижение веса</p>
```

Comment:
```suggestion
              <p class="live-example-plate__text">снижение веса</p>
```
не надо на будущее ставить тебе модификаторы или делать миксы из блока и элемента
как придешь к проблеме, что тебе необходим модификатор, так добавишь
преждевременная оптимизация😈